Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -0,0 +1,229 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay;
+block Controller
+  "Output relay signals for tuning PID controllers"
+  parameter Real yHig(
+    final min=1E-6) = 1
+    "Higher value for the relay output";
+  parameter Real yLow(
+    final min=1E-6,
+    final max=yHig) = 0.5
+    "Lower value for the output";
+  parameter Real deaBan(min=1E-6) = 0.5
+    "Deadband for holding the output value";
+  parameter Boolean reverseActing=true
+    "Set to true for reverse acting, or false for direct acting control action";
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput u_s
+    "Setpoint input signal"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+    iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput u_m
+    "Measurement input signal"
+    annotation (Placement(transformation(origin={0,-120},extent={{20,-20},{-20,20}},rotation=270),
+    iconTransformation(extent={{20,-20},{-20,20}},rotation=270,origin={0,-120})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput trigger
+    "Connector for enabling the relay controller"
+    annotation (Placement(transformation(extent={{-20,-20},{20,20}},rotation=90,origin={-80,-120}),
+    iconTransformation(extent={{-20,-20},{20,20}},rotation=90,origin={-60,-120})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput y
+    "Control output"
+    annotation (Placement(transformation(extent={{100,30},{140,70}}),
+        iconTransformation(extent={{100,40},{140,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yOn
+    "Relay switch output, true when control output switches to the higher value"
+    annotation (Placement(transformation(extent={{100,-80},{140,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yErr
+    "Control error"
```

Comment:
in the code implementation, the `yErr = u_m - u_s`, but in the documentation, the `yErr` could be `u_m - u_s` or `u_s - u_m`. Would you please confirm if the one in the code should be the same as described in the documentation? If it should be different, then we need to differentiate them.